src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -0,0 +1,388 @@
+package net.maizegenetics.phgv2.cli
+
+import biokotlin.genome.*
+import biokotlin.seq.NucSeq
+import biokotlin.seqIO.NucSeqIO
+import biokotlin.util.bufferedReader
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.validate
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.vcf.VCFAltHeaderLine
+import htsjdk.variant.vcf.VCFHeaderLine
+import htsjdk.variant.vcf.VCFHeaderVersion
+import net.maizegenetics.phgv2.utils.Position
+import net.maizegenetics.phgv2.utils.createHVCFRecord
+import net.maizegenetics.phgv2.utils.exportVariantContext
+import net.maizegenetics.phgv2.utils.getChecksumForString
+import java.io.File
+
+class CreateMafVcf : CliktCommand(help = "Create gVCF and hVCF from Anchorwave MAF files") {
+
+    val bed by option(help = "BED file")
+        .default("")
```

Comment:
Still seems like putting required() would be sufficant?

Reply:
I agree, but I have tried this and it does not work unless the default is there.